compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/BaseComposeScene.skiko.kt

```diff
@@ -149,7 +149,13 @@ internal abstract class BaseComposeScene(
         recomposer.performScheduledRecomposerTasks()
     }
 
-    override fun render(canvas: Canvas, nanoTime: Long) =
+    override fun render(canvas: Canvas, nanoTime: Long) {
+        // This is a no-op if the scene is closed, this situation can happen if the scene is
+        // in the list for rendering, but recomposition in another scene from the same list
+        // processed earlier has closed it.
+
+        if (isClosed) return
```

Comment:
Does it make sense to guard on platform bindings side? For the comment above it seems like a highly platform-specific situation

Reply:
`isClosed` is not exposed, what do you propose?

Reply:
Right, it's public. It's easier to keep it as is then. nvm